uxarray/__init__.py

```diff
@@ -1,2 +1,14 @@
 from .grid import *
 from .helpers import *
+# Sets the version of uxarray currently installed
+# Attempt to import the needed modules
+try:
+    from importlib.metadata import version as version
+except ImportError:
+    from importlib_metadata import version as version
```

Comment:
```suggestion
except Exception:
    from importlib_metadata import version as version
```

Again, I think it'd be better to catch as broadly as possible here, since it doesn't really matter why the first import fails.

You're right, it'll likely be an import error, but there's no reason to not go through the same steps if it's another kind of exception.

Reply:
Good point! Thanks for all the comments, the time is greatly appreciated!